python/tvm/relax/vm.py

```diff
@@ -583,6 +583,9 @@ def foo(x: Tensor((3, 4), "float32"), y: Tensor((3, 4), "float32")):
 
     passes = [relax.transform.ToNonDataflow()]
     passes.append(relax.transform.CallTIRRewrite())
+    if target.kind.name == "cuda":
```

Comment:
As Relax keeps a minimal building path, I'm not sure if it's reasonable to add cuda graph passes directly in VM build. cc @tqchen

Reply:
Ideally no, in this case because the CUDAGraph was mainly after CallTIRRewrite, it might make sense for now. We can consider to get some of these stages out of the build